tests/fixtures/TagLib.Tests.FileFormats/OpusFormatTest.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using NUnit.Framework;
+using TagLib;
+
+namespace TagLib.Tests.FileFormats
+{
+    [TestFixture]
+    public class OpusFormatTest : IFormatTest
+    {
+        private static string sample_file = "samples/sample.opus";
```

Comment:
I did add it to the pull request, it's the [new file](https://github.com/lesderid/taglib-sharp/blob/e7ad63dc92b398b90962e4ac3547de997d1666ac/tests/samples/sample.opus) in my [last commit](https://github.com/lesderid/taglib-sharp/commit/e7ad63dc92b398b90962e4ac3547de997d1666ac).
Perhaps you didn't notice it because it's a binary file?